### PR TITLE
Create README

### PR DESCRIPTION
**Pull request by apelisse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+[![Build Status](https://travis-ci.org/kubernetes/utils.svg?branch=master)](https://travis-ci.org/kubernetes/utils)
+
+Utils is a set of golang libraries that are not specific to
+Kubernetes. They should be available and useful to any other Go project
+out there.
+
+## Purpose
+
+The Kubernetes project uses a lot of Golang patterns that are re-used in
+many difference places, and since the project is more and more splitted
```
Comment (pwittrock) [inline README.md]: > since the project is more and more splitted

and as the project is further split into different

File: README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Go libraries in this repository must be:
+
+- Generic enough that they are useful for external/non-kubernetes
+  projects,
+- Well factored, well tested and reliable,
+- Be completely go compliant (go get/build/test/etc)
+- Have enough complexity to be shared,
+- Have stable APIs, or backward compatible.
+
+Even though the name of the repository is pretty generic, the goal is to
```
Comment (pwittrock) [inline README.md]: Just lead with `the goal is...`

**Comment by apelisse** [comment]
Updated!

**Pull request merged**